packages/pipeline-editor/src/utils.ts

```diff
@@ -105,4 +105,15 @@ export default class Utils {
 
     return widget;
   };
+
+  /**
+   * Strips the `elyra_` prefix from a property key if it exists
+   */
```

Comment:
I think it would be better to state _why_ this is necessary.  The code and method name already do a good job of _what_ is happening.

I found @kiersten-stokes' https://github.com/elyra-ai/elyra/pull/1986#discussion_r698831913 helpful.